boot_rom/Xcodes.h

```diff
@@ -287,8 +287,17 @@
 	xcode_pciout(0x80010010, 0xfd000000);
 	
 	
-	// overflow trick
-	xcode_poke(0x00000000, 0xfc1000ea);
-	xcode_poke(0x00000004, 0x000008ff);
+	// overflow trick (orig)
```

Comment:
* Let's remove this old commented-out code; git preserves the history for us, no need to keep it.
* Not necessary to mention xemu-friendliness. However, I would prefer to include a note which could help readers follow control flow, like:
```c
// overflow trick (execution continues in 2bBootStartup.S)
```